demo/style.css

```diff
@@ -29,93 +29,100 @@ pre {
 }
 
 code {
+  display: block;
   font-family: Consolas, Monaco, monospace;
   font-size: 10pt;
+  overflow-x: auto;
+  padding: 0.5em;
   scrollbar-width: thin;
+  -moz-tab-size: 4;
+  tab-size: 4;
```

Comment:
Where are we using tabs that these needed to be added?

Reply:
1c, autoit, crmsh, dts, gml, hsp, makefile, roboconf, sqf, tp

For example:
https://github.com/highlightjs/highlight.js/blob/3a89dfb51006751b34501ea4f78d744b6d98e91a/test/detect/1c/default.txt#L3

Reply:
Ok, I was not aware of this.  And how did we decide on 4 as the proper tab width?  What is the default?

Reply:
I wonder if those languages prefer/use tabs over spaces or if this is just a legacy oddity... This is a spaces codebase and my first urge was to go and change them all to spaces. :)

Reply:
I just picked 4 as it tends to look better than 8 and seems to usually be the default in most editors.